cp1616/src/cp1616_io_controller.cpp

```diff
@@ -448,6 +467,47 @@ void Cp1616IOController::setSemModChange(int mod_change)
   sem_mod_change_ = mod_change;
 }
 
+PNIO_UINT32 Cp1616IOController::parseConfigFile(std::string filepath)
+{
+  std::ifstream fin(filepath.c_str());
+  YAML::Parser parser(fin);
+  YAML::Node doc;
+  PNIO_UINT32 error_code = PNIO_OK;
+  
+  try
+  {
+    parser.GetNextDocument(doc);
+      
+    for(unsigned i = 0; i < doc.size(); i++)
+    {
+      ControllerModuleData temp_module;
+      doc[i] >> temp_module; 
+      if(temp_module.type == "input")
```

Comment:
Header
`static const std::string INPUT;`
`static const std::string OUTPUT;`

Cpp 
`const std::string Cp1616IOController::INPUT = "input";`
`const std::string Cp1616IOController::OUTPUT = "output";`

or is there any more elegant solution to this?